.github/workflows/create-release.yml

```
@@ -45,23 +65,52 @@ jobs:
         echo "# Platform-specific torch requirements (See SCT Issue #2745)" >> requirements-freeze.txt
         grep "torch" requirements.txt >> requirements-freeze.txt
 
-    - name: Update version.txt
+    - name: Update version.txt (using test value for CI)
+      run: |
+        echo "0.0" > spinalcordtoolbox/version.txt
+      # Only attach to release if workflow is run manually. (This allows the workflow to double as a PR test.)
```

Since this condition is the opposite of the condition below, I assume the comment should also be the opposite? Or else I don't quite understand the phrase "only attach to release".
```suggestion
      # Only attach to release if workflow is run automatically. (This allows the workflow to double as a PR test.)
```

---

Good catch! This was a copy-paste that I forgot to update. :sweat_smile: 

In fact, we're not even attaching the file to the release at all, so the whole comment is misleading. It would probably be better as something like:

```suggestion
      # If the workflow *hasn't* been run manually, then it's being run as a PR test, so use a dummy value instead
```